workshops/Tkinter/README.md

```diff
@@ -0,0 +1,135 @@
+---
+name:'Make User Interfaces for your python code.'
+description:'We will use python tkinter graphics to make User Interfaces. Its a nice world.'
```

Comment:
could this describe what you are actually making?

Reply:
yeah
in this tutorial we are making a interface for our python code

Reply:
what is the interface though.... as a kid I hear make an interface and im like mhhh boring

Reply:
oh let me think one

Reply:
Make design rather than just black and white lines for your python codes